src/panels/config/zha/zha-clusters.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  PropertyValues,
+} from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-card/paper-card";
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import "../../../components/buttons/ha-call-service-button";
+import "../../../components/ha-service-description";
+import "../ha-config-section";
+
+import { HomeAssistant } from "../../../types";
+import "../../../resources/ha-style";
+import { ItemSelectedEvent, Cluster, ZHADeviceEntity } from "./types";
+import { fireEvent } from "../../../common/dom/fire_event";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "zha-cluster-selected": {
+      cluster?: Cluster;
+    };
+  }
+}
+
+export class ZHAClusters extends LitElement {
+  public hass?: HomeAssistant;
+  public isWide?: boolean;
+  public showHelp: boolean;
+  public selectedEntity?: ZHADeviceEntity;
+  private _selectedClusterIndex: number;
+  private _clusters: Cluster[];
+  private _haStyle?: DocumentFragment;
+  private _ironFlex?: DocumentFragment;
+
+  constructor() {
+    super();
+    this.showHelp = false;
+    this._selectedClusterIndex = -1;
+    this._clusters = [];
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      isWide: {},
+      showHelp: {},
+      selectedEntity: {},
+      _selectedClusterIndex: {},
+      _clusters: {},
+    };
+  }
+
+  protected update(changedProperties: PropertyValues): void {
+    if (changedProperties.has("selectedEntity")) {
+      this._clusters = [];
+      this._selectedClusterIndex = -1;
+      fireEvent(this, "zha-cluster-selected", {
+        cluster: undefined,
+      });
+      this._fetchClustersForZhaNode();
+    }
+    super.update(changedProperties);
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      ${this._renderStyle()} ${this._renderClusterPicker()}
```

Comment:
Remove `renderClustPicker` and just put it in the `render` function

Reply:
yeah this one I would inline